core/cmd/eth_keys_commands_test.go

```diff
@@ -182,7 +182,7 @@ func TestShell_CreateETHKey(t *testing.T) {
 		withKey(),
 		withMocks(ethClient),
 	)
-	db := app.GetSqlxDB()
+	db := app.GetDB()
```

Comment:
many other callsites where renames to DS. why is this DB? is there a distinction btw the two or are we intermixing naming conventions?

Reply:
I'm not opposed to aligning things a bit more but was trying to limit thrashing in places I wasn't already touching. 
There are arguably places where `db` or `conn` or `tx` are appropriate though, since that is all there will be, butwe have wrapped the particular type up as a `DataSource` to apply hooks, so we lose that information.